src/shapes/Text/constants.ts

```diff
@@ -8,7 +8,7 @@ const fontProperties = [
   'fontStyle',
 ] as const;
 
-const textDecorationProperties = [
+export const textDecorationProperties = [
```

Comment:
do we need to export this?

Reply:
I think maybe is a leftover of trying to get the type automatically from this array?

Reply:
Something like that
I used it for the HTML logic

Reply:
Not really